packages/graphback-db-migrate/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@graphback/db-migrate",
+  "version": "0.10.0-dev1",
+  "description": "Graphback database migrations",
+  "main": "dist/index.js",
+  "types": "types/index.d.ts",
+  "typescript": {
+    "definition": "types/index.d.ts"
+  },
+  "dependencies": {
+    "@graphback/codegen-core": "0.10.0-dev1",
+    "@graphql-inspector/core": "1.26.0",
+    "glob": "7.1.5",
+    "knex": "0.20.0",
+    "pino": "5.13.5"
+  },
+  "devDependencies": {
+    "@types/graphql": "14.2.3",
+    "@types/knex": "0.15.2",
+    "@types/node": "8.10.56",
+    "ava": "2.4.0",
+    "graphql": "14.5.8",
```

Comment:
Nothing in the package uses graphql.

Reply:
It does: https://github.com/aerogear/graphback/blob/2344719014711a1beab755dc487029252f85c401/packages/graphback-db-migrate/src/database/migrations/schema/GraphQLSchemaManager.ts#L2